packages/stylist/src/yew/global.rs

```diff
@@ -40,24 +40,29 @@ pub struct GlobalProps {
 pub fn global(props: &GlobalProps) -> Html {
     let mgr = use_context::<StyleManager>().unwrap_or_default();
 
-    #[derive(Debug, PartialEq)]
-    struct GlobalDependents {
-        manager: StyleManager,
-        css: StyleSource,
+    #[cfg(feature = "ssr")]
+    {
```

Comment:
Could this not be the universal implementation, with a wrapper struct that has a `Drop` impl that unregisters the `GlobalStyle`?
If the client is compiled with feature `ssr` enabled (which could due to feature unification in workspaces, or just by mistake), this always leaks a style. Might as well `use_memo( ... , (mgr, props.css.clone()),);` here for all codepaths. `use_style` also immediately mounts the style, instead of in an effect.

Reply:
> this always leaks a style

Because effect is called with `unmount()` in the destructor, it will unmount regardless of whether there are other existing instance. So it will unmount correctly.

> Might as well use_memo( ... , (mgr, props.css.clone()),); here for all codepaths. use_style also immediately mounts the style, instead of in an effect.

Sounds good, I will make the change.